src/app/ApplicationTemplate.Shared.Views/Styles/Controls/CommandBar.xaml

```diff
@@ -86,15 +92,19 @@
 
 		<Setter Property="HorizontalAlignment"
 				Value="Stretch" />
+
```

Comment:
Why add all those blank line?

Reply:
Making it more readable, I was finding it hard to read everything while updating this. Also not that much code so I thought it would be too much having some spacing.